src/Adapter/MSTest.TestAdapter/Execution/TestMethodInfo.cs

```diff
@@ -260,6 +263,20 @@ public virtual TestResult Invoke(object?[]? arguments)
         return expectedExceptions.FirstOrDefault();
     }
 
+    /// <summary>
+    /// Gets the number of retries this test method should make in case of failure.
+    /// </summary>
+    /// <returns>
+    /// The number of retries, which is always greater than or equal to 1.
+    /// If RetryAttribute is not present, returns 1.
```

Comment:
I guess it's how you interpret language, but in TUnit, lack of a retry attribute means a retry count of 0;

Because if I do `[Retry(1)]` I'd expect it to try again after a failure, once.

If I'm reading this PR correctly, `[Retry(1)]` currently means, run the test normally as you would and never retry? So that's retrying 0 times.

Does that make sense? The number doesn't work with the wording. This usage would work better with `[Attempt(1)]` or `[Try(1)]` or something.

Reply:
@thomhurst Yup indeed. I'll be working on a fix so that `[Retry(n)]` means it can run `1 + n` times at max (normal first attempt + n retries).